boards/arm/96b_carbon/96b_carbon.dts

```diff
@@ -43,6 +43,13 @@
 
 &spi1 {
 	status = "ok";
+
+	nrf51822-qfac@0 {
+		compatible = "nordic,nrf51822-qfac";
```

Comment:
The compatible here isn't correct.  Saying "nordic,nrf51822-qfac" isn't specific enough.  We need something that conveys the programming interface/driver one would select for the "device" connected.  In practice, this seems like it would instead be something like "zephyr,bt-spi" (but that's just a guess).

Reply:
In case it's relevant, this is the sample implementing the SPI protocol (as used on carbon): http://docs.zephyrproject.org/samples/bluetooth/hci_spi/README.html

Reply:
I was very skeptical when I added nordic,nrf51822-qfac.yaml file because it is not only a bluetooth module. I agree we need to change it to zephyr,bt-hci-spi.yaml, it would make clear which driver is used when someone reads boards dts file. I will do the changes.

Reply:
> I was very skeptical when I added nordic,nrf51822-qfac.yaml file because it is not only a bluetooth module. I agree we need to change it to zephyr,bt-hci-spi.yaml, it would make clear which driver is used when someone reads boards dts file. I will do the changes.

I was not satisfied by the above approach because nrf51822-qfac wasn't mentioned anywhere in the device tree file of 96b_carbon board. I moved back to nordic,nrf51822-qfac.yaml and used compatible property to specify the driver used by the module.